plugins/modules/net_tools/nmcli.py

```diff
@@ -95,6 +95,47 @@
         type: list
         elements: str
         version_added: 2.0.0
+    routes4_extended:
+        description:
+            - The list of ipv4 routes.
+        type: list
+        elements: dict
+        suboptions:
+            ip:
+                description:
+                    - ip or prefix of route.
+                    - Use the format '192.0.3.0/24'.
+                type: str
+                required: true
+            next_hop:
+                description:
+                    - Use the format '192.0.2.1'.
```

Comment:
There should be some description as well.
```suggestion
                    - Use the format C(192.0.2.1).
```